cmd/microcloud/ask.go

```diff
@@ -1179,12 +1186,13 @@ func (c *initConfig) askOVNNetwork(sh *service.Handler) error {
 					return err
 				}
 
-				ovnUnderlaySelectedIPs = map[string]string{}
+				ovnUnderlaySelectedIPs = map[string]*ipWithIface{}
```

Comment:
The map already got initialized a few lines before so I think we don't even needs this line?

Reply:
I can remove the initialization at line 1177 and initialize it here instead. The thing is that it needs to be initialized inside the callback of askRetry in case of error (the user did not choose the right amount of IPs). If there is no reset of this map, it'll be populated with previous choices of the user.